### DISCUSSION TITLE
Scenario Outline name is missing

### DISCUSSION THREAD

**Discussion by Helene-byte** [allure-framework/allure2#2089] [Questions & Support]
i am using Cucumber 7.10.0 + junit5 + allure 2.10. Scenario names look perfectly fine in allure report, though scenario outline name is missing. 
<img width="1438" alt="Screenshot 2023-08-25 at 15 32 56" src="https://github.com/allure-framework/allure2/assets/51850053/381c39cb-a820-4992-ba53-ffbdb393ce3f">

As soon as scenario is changed to scenario outline the name disappears and the report looks as on the screen above.
Would appreciate any ideas , thank you!

**Comment by baev** [comment]
Allure 2.10 was released 4.5 years ago. Please update to the latest version (2.23.0 as for today)

**Reply by Helene-byte** [reply]
hi @baev , thank you for your response! changed but does not work so far - 

on the screen below there is a report generated for scenario outline with 2 examples and scenario. Scenario name is shown though Scenario outline name is missing :(

<img width="1353" alt="Screenshot 2023-08-26 at 10 23 59" src="https://github.com/allure-framework/allure2/assets/51850053/5563ab71-d380-40ef-839f-39d0e67ae829">


Cucumber built-in reporter works as expected (apart from number of tests cases in the header but that is the other story) - Scenario outline name is shown as well as Scenario:


<img width="1373" alt="Screenshot 2023-08-27 at 09 24 31" src="https://github.com/allure-framework/allure2/assets/51850053/327b8f54-bef0-4a59-8ae6-126ce2ed3837">


Could you please, take a look in a below post with dependencies versions ? Would you think this should work as expected ?

**Comment by Helene-byte** [comment]
Hi @baev , hope you are doing great!  Thank you for your prompt response. i tried with 2.23.0 with no success so far. Would you consider these version dependencies to work as expected ? 
<img width="722" alt="Screenshot 2023-08-26 at 08 54 54" src="https://github.com/allure-framework/allure2/assets/51850053/56066c1b-ebbd-4a3d-a43c-f3578d168aed">
Thank you!

**Reply by baev** [reply]
Could you please provide me with a sample project (a GitHub repo) that reproduces the problem?

**Reply by Helene-byte** [reply]
Hi @baev ,hope you are doing great. Please, find the link attached https://github.com/oKrasyl/playwright-java-cucumber

**Reply by Helene-byte** [reply]
Could you have a look @baev  <img width="801" alt="Screenshot 2023-09-17 at 19 20 11" src="https://github.com/allure-framework/allure2/assets/51850053/e1aec358-a087-44f3-928c-be490eb2c375">

**Reply by Helene-byte** [reply]
There is a possibility to include the feature name as part of the tests name as it is specified on https://github.com/cucumber/cucumber-jvm/tree/main/cucumber-junit-platform-engine#use-the-junit-platform-suite-engine. Would you think there is more elegant way to change this ? Thank you!

**Reply by Helene-byte** [reply]
Hi @baev , hope you are doing great ) would you have a chance to look into my query ? Thank you!

**Reply by baev** [reply]
Sorry for the delay. I'll include this in my backlog for the next week.

**Comment by baev** [comment]
I can't reproduce the problem (or I didn't get it correctly)

I followed the steps: 

1. https://allurereport.org/start/
2. Select Java -> CucumberJVM
3. Keep defaults, download the project
4. Run `./mvnw verify` to run tests
5. Build report using `allure serve target/allure-results` (latest version of allure is installed)
    <img width="1546" alt="Screenshot 2024-02-06 at 12 57 48" src="https://github.com/allure-framework/allure2/assets/2149631/f1f1ad2b-161f-4dc6-b909-8f948d83ae1c">
7. Modified scenario `src/test/resources/features/labels-rest.feature`
    ```feature
    @smoke
    Scenario Outline: Create new label via api
      When I create label with title "hello" via api
      Then I should see label with title "hello" via api
      Examples:
        |a  |
        |1  |
    ```
8. Scenario name is displayed correctly, and parameters are shown as expected
    <img width="1546" alt="Screenshot 2024-02-06 at 12 53 22" src="https://github.com/allure-framework/allure2/assets/2149631/9752729a-5ee4-4243-a0a8-58efe6e4de6d">

**Reply by Helene-byte** [reply]
hi @baev thank you for your feedback, but i am afraid the features on the screen do not belong to my suite ,or i may probably be missing something :(

**Reply by Helene-byte** [reply]
i have cloned the project from repository mentioned above and mvn clean test.
 the link attached https://github.com/oKrasyl/playwright-java-cucumber 
 this is the report i received : 
<img width="877" alt="Screenshot 2024-02-06 at 16 14 56" src="https://github.com/allure-framework/allure2/assets/51850053/8fa4fbbd-16f9-4486-8998-8dc71d811880">

**Reply by baev** [reply]
The project you referred to contains numerous configurations, and I can't run it because I don't have a Chrome browser. Either way, please follow the steps I outlined above and confirm if they work for you. Then, you will have a functioning example and a project with issues. This allows you to compare configurations step by step and attempt to identify the problem.

**Reply by Helene-byte** [reply]
thank you @baev  , got it, would investigate for differences. within this configuration it works as it would be expected.
